keyboards/cmm.studio/saka68/readme.md

```diff
@@ -0,0 +1,12 @@
+# CMM.Studio Saka68
```

Comment:
As above.

Reply:
deleted the whole folder of cmm.studio. now only have cmm_studio folder